app/styles/globals.scss

```diff
@@ -28,6 +28,22 @@ body {
   transition: background-color 0.3s ease-in-out;
 }
 
+// Hidden utilities
```

Comment:
This gives us the ability to do `className="hidden s:visible"` of vice versa. That's pretty handy IMO - a couple of examples in this PR!

The downsides of this approach are:
- We only have access to "display: unset;" - this works for this use case but I don't think is a full solution
- This is a weird place to put hidden styling - it does _not_ play well with our `flex` className for example
- Hidden elements still render (have React lifecycles), they just don't appear in the dom

The alternative solution is to create a component that listens to the page width and applies a media query - the problem with this is it can only run on the client. Given we're server-side rendering, I think we need a CSS based solution too 🙏